oauth2/fosite_store_sql.go

```diff
@@ -355,3 +363,53 @@ func (s *FositeSQLStore) FlushInactiveAccessTokens(ctx context.Context, notAfter
 
 	return nil
 }
+
+// internal convenience interface to keep storage methods DRY otherwise there will be hella lot of copy-pasta
+// note sure if this is the best way to go about this tho ¯\_(ツ)_/¯
+type operatorExtContext interface {
+	sqlx.ExtContext
+	GetContext(ctx context.Context, dest interface{}, query string, args ...interface{}) error
+	NamedExecContext(ctx context.Context, query string, arg interface{}) (sql.Result, error)
+}
+
```

Comment:
@aeneasr what do you think of 🔝?

It's basically a means to avoid copy-pasta hell across all the storage methods.

Reply:
LGTM, if it's unexported that's totally fine IMO